src/DonationForms/resources/components/Pagination.js

```diff
@@ -19,86 +20,89 @@ const Pagination = ({currentPage, totalPages, disabled, setPage}) => {
     const previousPage = parseInt(currentPage) - 1;
 
     return (
-        <div className="tablenav bottom">
-            <div className="tablenav-pages">
-                <div className="pagination-links">
-                    {previousPage > 0 ? (
-                        <>
-                            <a
-                                href="#"
-                                className="tablenav-pages-navspan button"
-                                onClick={(e) => {
-                                    e.preventDefault();
-                                    if (!disabled) {
-                                        setPage(1);
-                                    }
-                                }}
-                            >
-                                «
-                            </a>{' '}
-                            <a
-                                href="#"
-                                className="tablenav-pages-navspan button"
-                                onClick={(e) => {
-                                    e.preventDefault();
-                                    if (!disabled) {
-                                        setPage(parseInt(currentPage) - 1);
-                                    }
-                                }}
-                            >
-                                ‹
-                            </a>
-                        </>
-                    ) : (
-                        <span className="tablenav-pages-navspan button disabled">‹</span>
-                    )}
+        <nav className={styles.container}>
```

Comment:
Since there is already an element with `role=navigation` on the page (a part of the WordPress admin), we should add an `aria-label` attribute with a meaningful name for this.

```suggestion
        <nav aria-label={__('donation forms table', 'give')} className={styles.container}>
```

Screen readers announce “navigation” after the label, so this would read “donation forms table navigation.”